source/manual/setting-up-new-rails-app.html.md

```diff
@@ -8,140 +8,161 @@ last_reviewed_on: 2018-03-22
 review_in: 6 months
 ---
 
-Getting a new Rails app running in production has several steps.
-
-You should ensure that:
+[govuk-puppet]: https://github.com/alphagov/govuk-puppet/blob/master/docs/adding-a-new-app.md#including-the-app-on-machines
+[govuk-puppet-jenkins]: https://github.com/alphagov/govuk-puppet/blob/master/hieradata/common.yaml
+[dns]: https://docs.publishing.service.gov.uk/manual/dns.html#making-changes-to-publishingservicegovuk
+[govuk-saas-config]: https://github.com/alphagov/govuk-saas-config
+[govuk-app-deployment]: https://github.com/alphagov/govuk-app-deployment
+[sentry]: https://sentry.io/settings/govuk/teams/
+[release]: https://release.publishing.service.gov.uk/applications
+[deploy-jenkins]: https://deploy.integration.publishing.service.gov.uk/job/Deploy_App/
```

Comment:
Is convention not to have this at the bottom?

Reply:
I've mainly seen it at the top ;-)